portal-api/src/v1/users/controller.js

```diff
@@ -138,35 +139,37 @@ exports.findByEmail = async (email, callback) => {
 
 exports.create = async (user, callback) => {
   const insert = {
-    'user-status': ACTIVE,
-    timezone: user.timezone || 'Europe/London',
+    'user-status': USER.STATUS.ACTIVE,
+    timezone: USER.TIMEZONE.DEFAULT,
```

Comment:
Sorry - Lack of context here on my behalf. We're happy to have all user timezones to be the default?

Reply:
Yes that's correct.